### ISSUE TITLE
Usuario: Recuperar contraseña

### ISSUE THREAD

**Issue by JohanaSfC** [ScienceForChange/odourCollect_2.0#210]
En la pantalla se describe que la contraseña debe ser diferente a las anteriores. Entiendo como medida de seguridad.

Pero no está funcionando correctamente, estaba probando el botón de recuperar contraseña y coloqué mi contraseña inicial y no me generó ningún error. Es decir, el USUARIO PUEDE TENER UNA CONTRASEÑA IGUAL A LAS ANTERIORES. 

Por favor, si esta lógica es correcta o si hay algún fallo en el código. @CarlosMillanSfC @OtariOboladzeSfC 

<img width="146" alt="image" src="https://github.com/ScienceForChange/odourCollect_2.0/assets/83076037/31cee094-658b-4d50-915c-eedd2d433fb3">

**Comment by almenarasfc** [comment]
Esta por definir que esta pantalla esté en la App, quizá forme parte del backend, todavía no podemos probar como se enlaza el email de recuperar contraseña con la App. Al menos en iOS.

**Comment by SysAdminSfC** [comment]
@almenarasfc  puedes borrar el texto "debe ser diferente" porfa? no hace falta, los usuarios pueden poner la misma contraseña si quieren. @CarlosMillanSfC me puedes confirmar que es asi? que no hay ninguna logica que impide usuarios poner la misma contraseña?

**Comment by CarlosMillanSfC** [comment]
@SysAdminSfC No, no hay ninguna lógica que compruebe que la contraseña sea diferente, yo también creo que lo mejor es eliminar ese texto y si el usuario quiere poner la misma que la ponga.

Cuando @almenarasfc borre esa parte que cierre esta issue porfa.

**Comment by CarlosMillanSfC** [comment]
Lo subo yo, quitamos ese texto y listo.
